delira/training/predictor.py

```diff
@@ -547,8 +627,36 @@ def calc_metrics(batch: LookupConfig, metrics=None, metric_keys=None):
         if metrics is None:
             metrics = {}
         if metric_keys is None:
-            metric_keys = {k: ("pred", "label") for k in metrics.keys()}
+            metric_keys = {k: ("label", "pred") for k in metrics.keys()}
 
         return {key: metric_fn(*[batch.nested_get(k)
                                  for k in metric_keys[key]])
                 for key, metric_fn in metrics.items()}
+
+    def register_callback(self, callback: AbstractCallback):
+        """
+        Register Callback to Trainer
+
+        Parameters
+        ----------
+        callback : :class:`AbstractCallback`
+            the callback to register
+
+        Raises
+        ------
+        AssertionError
+            `callback` is not an instance of :class:`AbstractCallback` and has
+            not both methods ['at_epoch_begin', 'at_epoch_end']
```

Comment:
Wrong fucntions in docstring: function checks if `at_iter_being/end` are present.